cypress/utils/setup-utils.js

```diff
@@ -35,7 +35,7 @@ const createAndVerifyOrganization = (orgName) => {
 const archivePatientsForFacility = (facilityId) => {
   return getPatientsByFacilityId(facilityId).then((res) => {
     let patients = res.body.data.patients;
-    if (patients.length > 0) {
+    if (patients && patients.length > 0) {
```

Comment:
we were running into some null-pointer edge cases in local testing so went ahead and added guards for all these variables in this file. Wanted to use the `?.` operator, but that made our eslint setup made bc I think we aren't on the right ECMAScript version for our linting, so just doing this the old-school javascript way instead.